quickstart/201-iot-hub-with-device-provisioning-service/main.tf

```diff
@@ -0,0 +1,165 @@
+resource "random_pet" "rg_name" {
+  prefix = var.resource_group_name_prefix
+}
+
+resource "azurerm_resource_group" "rg" {
+  location = var.resource_group_location
+  name     = random_pet.rg_name.id
+}
+
+# Create storage account & container
+resource "random_string" "sa_name" {
+  length = 12
```

Comment:
Could we run `terraform fmt` command for this directory to align these equal marks please?